src/zenml/zen_stores/schemas/step_run_schemas.py

```diff
@@ -312,7 +312,7 @@ class StepRunInputArtifactSchema(SQLModel, table=True):
 
     # Fields
     name: str = Field(nullable=False, primary_key=True)
-    type: StepRunInputArtifactType
```

Comment:
Just commenting as a reminder for myself to check that this is handled correctly in wherever this is used.